src/__init__.py

```diff
@@ -3,7 +3,13 @@
 
 Contains saga formatter and function to create a logger.
 """
+from .__about__ import (
+    __author__, __copyright__, __email__, __license__, __summary__,
+    __title__, __uri__, __version__
+)
 from .formatter import SagaFormatter
 from .creator import get_logger
 
-__all__ = [SagaFormatter, get_logger]
+__all__ = [
+    SagaFormatter, get_logger, __author__, __copyright__, __email__,
+    __license__, __summary__, __title__, __uri__, __version__]
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Undefined variable name 'MIT' in __all__ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=Pylint%3AE0603)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Undefined variable name 'A json log formatter.' in __all__ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=Pylint%3AE0603)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Undefined variable name 'sagalogger' in __all__ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=Pylint%3AE0603)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Undefined variable name 'https://www.github.com/Sagacify/logger-py' in __all__ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=Pylint%3AE0603)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Undefined variable name '0.3.0' in __all__ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=Pylint%3AE0603)